modules/affile/logproto-file-writer.c

```diff
@@ -134,6 +135,14 @@ log_proto_file_writer_flush(LogProtoClient *s)
   self->buf_count = 0;
   self->sum_len = 0;
 
+  GString *request_body = g_string_new("");
+  g_string_append_printf(request_body, "%d", self->buf_size);
+  GString *file_name = g_string_new("");
+  g_string_append_printf(file_name, "%s", self->super.transport->name);
```

Comment:
`request_body` and `file_name` are not used currently. Let's remove them for now.

Reply:
Updated 👍